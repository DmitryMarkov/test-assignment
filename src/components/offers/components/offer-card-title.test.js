import React from 'react'
import { render } from '@testing-library/react'
import OfferCardTitle from './offer-card-title'

test('OfferCardTitle should render properly', () => {
  const mockTitle = 'Grand Royal'
  const { asFragment, getByText } = render(<OfferCardTitle title={mockTitle} />)

  expect(getByText(mockTitle)).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot()
})
