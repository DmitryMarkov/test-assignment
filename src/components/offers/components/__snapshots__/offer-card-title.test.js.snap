// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferCardTitle should render properly 1`] = `
<DocumentFragment>
  <h3
    class="css-50tfpm"
  >
    Grand Royal
  </h3>
</DocumentFragment>
`;
